Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -294,6 +294,55 @@ public void EvaluateWithPolicyActions()
                     .GetState(_endTxValueAddress));
         }
 
+        [Fact]
+        public void EvaluateWithPolicyActionsWithException()
+        {
+            var store = new MemoryStore();
+            var stateStore = new TrieStateStore(new MemoryKeyValueStore());
+            var policyWithExceptions = new BlockPolicy(
+                beginBlockActions: new IAction[]
+                {
+                    new UpdateValueAction(_beginBlockValueAddress, 1),
+                    new ThrowException { ThrowOnExecution = true },
+                }.ToImmutableArray(),
+                endBlockActions: new IAction[]
+                {
+                    new UpdateValueAction(_endBlockValueAddress, 1),
+                    new ThrowException { ThrowOnExecution = true },
+                }.ToImmutableArray(),
+                beginTxActions: new IAction[]
+                {
+                    new UpdateValueAction(_beginTxValueAddress, 1),
+                    new ThrowException { ThrowOnExecution = true },
+                }.ToImmutableArray(),
+                endTxActions: new IAction[]
+                {
+                    new UpdateValueAction(_endTxValueAddress, 1),
+                    new ThrowException { ThrowOnExecution = true },
+                }.ToImmutableArray());
+
+            var (chain, actionEvaluator) =
+                TestUtils.MakeBlockChainAndActionEvaluator(
+                    policy: policyWithExceptions,
+                    store: store,
+                    stateStore: stateStore,
+                    actionLoader: new SingleActionLoader(typeof(DumbAction)));
+
+            (_, Transaction[] txs) = MakeFixturesForAppendTests();
+            var block = chain.ProposeBlock(
+                GenesisProposer, txs.ToImmutableList(), CreateBlockCommit(chain.Tip));
+            var evaluations = actionEvaluator.Evaluate(
+                block, chain.Store.GetStateRootHash(chain.Tip.Hash)).ToArray();
+
+            // BeginBlockAction + (BeginTxAction + #Action + EndTxAction) * #Tx + EndBlockAction
+            Assert.Equal(
+                4 + txs.Length * 4 + txs.Aggregate(0, (sum, tx) => sum + tx.Actions.Count),
```

Comment:
```suggestion
                4 + txs.Length * 4 + txs.SelectMany(item => item.Actions).Count(),
```
or

```suggestion
                4 + txs.Length * 4 + txs.Sum(item => item.Actions.Count),
```

I think it's better to use `Sum` instead of `Aggregate` because the test project targets the .NET 6.0 framework.